packages/ui/src/components/SnackBar/SnackToast.tsx

```diff
@@ -0,0 +1,88 @@
+import { useState, useEffect, useCallback } from "react";
+import { Alert, AlertTitle, Box, Button, Grow } from "@mui/material";
+import { useSnackStack, ToastProps } from "./SnackStackProvider";
+import LaunchIcon from '@mui/icons-material/Launch';
+import { styled }  from "@mui/material/styles";
+
+type SnackbarToastProps = {
+  toast: ToastProps;
+  className?: string;
+};
+
+const TIMEOUT = 300;
+
+const SnackbarToast: React.FC<SnackbarToastProps> = ({ toast, className }) => {
+  const [open, setOpen] = useState(true);
+  const { removeToast } = useSnackStack();
+
+  const close = useCallback(() => {
+    setOpen(false);
+    setTimeout(() => {
+      removeToast(toast.key);
+    }, TIMEOUT);
+  }, [toast.key, removeToast]);
+
+  const handleClose = (
+    event: React.SyntheticEvent | Event,
+    reason?: string
+  ) => {
+    if (reason === "clickaway") {
+      return;
+    }
+    if (toast?.onClose) {
+      toast.onClose();
+    }
+    close();
+  };
+
+  const onOpenLink = useCallback(() => {
+    window.open(toast.link, "_blank")
+  }, [toast.link])
+
+  useEffect(() => {
+    if (toast.duration !== 0) {
+      setTimeout(() => {
+        close();
+      }, toast.duration || 6000);
+    }
+  }, [close, toast.duration]);
+
+  return (
+    <Grow className={className} in={open} timeout={TIMEOUT}>
+      <Alert
```

Comment:
Can we please use Snackbar rather than Alerts? Or why do you use Alert? I prefer the look and feel of snackbar, it's more neutral than those loud colors.